src/components/AddMoney/components/AddMoneyBankDetails.tsx

```
@@ -11,7 +11,10 @@ import { useRouter, useParams } from 'next/navigation'
 import { useEffect, useState, useMemo } from 'react'
 import { countryCodeMap, countryData } from '@/components/AddMoney/consts'
 import { formatCurrencyAmount } from '@/utils/currency'
+import { formatBankAccountDisplay } from '@/utils/format.utils'
+import { getOnrampCurrencyConfig, getCurrencySymbol } from '@/utils/bridge.utils'
 import Icon from '@/components/Global/Icon'
+import { getCurrencySymbol, getOnrampCurrencyConfig } from '@/utils/bridge.utils'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getCurrencySymbol'. Consider to delete it or rename it.

'getCurrencySymbol' is defined here:


(lint/suspicious/noRedeclare)